docs/install.qmd

```diff
@@ -74,7 +74,7 @@ conda update -c conda-forge shiny
 In the same directory you created above, run:
 
 ```bash
-shiny create .
+shiny create
```

Comment:
```suggestion
shiny create -t basic-app -m express -d .
```

Reply:
Oh that's a good note, we have a huge amount of documentation about to ship on Express, but for this PR we should probably take it out. Thanks! 

For reference here's the Express documentation PR https://github.com/posit-dev/py-shiny-site